packages/docusaurus/src/webpack/utils.ts

```diff
@@ -326,8 +326,9 @@ type FileLoaderUtils = {
 
 // Inspired by https://github.com/gatsbyjs/gatsby/blob/8e6e021014da310b9cc7d02e58c9b3efe938c665/packages/gatsby/src/utils/webpack-utils.ts#L447
 export function getFileLoaderUtils(): FileLoaderUtils {
-  // files/images < 10kb will be inlined as base64 strings directly in the html
-  const urlLoaderLimit = 10000;
+  // files/images < 10kb (overridable via 'URL_LOADER_LIMIT' environment
+  // variable) will be inlined as base64 strings directly in the html
+  const urlLoaderLimit = process.env.URL_LOADER_LIMIT ?? 10000;
```

Comment:
Have you tested this locally? Afaik `process.env.URL_LOADER_LIMIT` would be a string, shouldn't it be converted to a number?

Reply:
I also suggest:
- moving it to `packages/docusaurus/src/constants.ts`
- name it `WEBPACK_ URL_LOADER_LIMIT`
- add comment to say it's temporary, link to this PR/issue

We'd rather centralize all those undocumented env variables in a single place

Reply:
> Have you tested this locally? Afaik `process.env.URL_LOADER_LIMIT` would be a string, shouldn't it be converted to a number?

It works with strings, but let's add that '+'.

I'll move the const and rename the ENV VAR as per your suggestion.

Reply:
Done.